Jint/Native/Object/ObjectInstance.cs

```diff
@@ -18,8 +19,7 @@ namespace Jint.Native.Object
 {
     public class ObjectInstance : JsValue, IEquatable<ObjectInstance>
     {
-        protected Dictionary<string, PropertyDescriptor> _intrinsicProperties;
-        protected internal Dictionary<string, PropertyDescriptor> _properties;
+        internal StringDictionarySlim<PropertyDescriptor> _properties;
```

Comment:
Why a single dictionary now? Didn't I do two dictionaries on purpose?

Reply:
Well I couldn't find the reason to have separate _intrinsicProperties, tried to read the ECMA spec and it doesn't have such distinction. Symbolic properties work the same way as normal properties etc.

Reply:
I think it might be for iterating. Pretty sure it was necessary.

Reply:
Let's see how it goes without it then, next time I think something matters I'll write a unit test ;)